Dockerfile

```diff
@@ -1,9 +1,11 @@
 FROM ubuntu:22.10
 
-RUN apt-get update && apt-get install -y python3.10.5 python3.10.5-dev
-
-COPY . .
+# Adding user
+RUN groupadd GraphGroup
```

Comment:
Without the system group, you'll be unable to modify the content of the files when running the app script.